deployment/pytorch/torch-hub-yolov5.ipynb

```diff
@@ -0,0 +1,905 @@
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "p7x7dfWoCNxK"
+      },
+      "outputs": [],
+      "source": [
+        "deployed_model = endpoint.get_deployed_model()"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 9. Predictions"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "4nS6rzzyK58c"
+      },
+      "outputs": [],
+      "source": [
+        "def process_image(url):\n",
+        "    return deployed_model.predict(url)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "gH_5iPC84jmu"
+      },
+      "outputs": [],
+      "source": [
+        "urls = [\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_0.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_1.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_2.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_3.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_4.jpg'],\n",
+        "    ['http://www.cs.ucf.edu/~aroshan/index_files/Dataset_PitOrlManh/images/000001_5.jpg']\n",
+        "]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "XOKZr5qBJqup"
+      },
+      "outputs": [],
+      "source": [
+        "results = []\n",
+        "start_time = time.time()\n",
+        "\n",
+        "with concurrent.futures.ThreadPoolExecutor() as executor:\n",
+        "    for result in urls, executor.map(process_image, urls):\n",
+        "        results.append(result)\n",
+        "        \n",
+        "end_time = time.time()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "pi546PBJJqj9"
+      },
+      "outputs": [],
+      "source": [
+        "total_time = end_time - start_time\n",
+        "total_time = time.strftime('%Hh %Mm %Ss', time.gmtime(total_time))"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "WSIMswsbW5Ic",
+        "outputId": "eb5c1585-64ae-45b6-d279-9575f12dfe90"
+      },
+      "outputs": [],
+      "source": [
+        "print(f\"Processing Time: {total_time} for {len(urls)} URLs.\")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "VeV_oVzddA0m"
+      },
+      "outputs": [],
+      "source": [
+        "df_results = []\n",
+        "\n",
+        "for item in results[1]:\n",
```

Comment:
If we make the change I mentioned in my other comment, this can just be

```suggestion
        "for item in results:\n",
```

Reply:
Stored variables made this work, and it shouldn't :sweat_smile: 
Deployed twice restarting the environment, now it's ok.